Sources/Foundation/Process.swift

```diff
@@ -1153,10 +1153,13 @@ open class Process: NSObject {
 
     // poll the runLoop in defaultMode until process completes
     open func waitUntilExit() {
-        
-        repeat {
-            
-        } while( self.isRunning == true && RunLoop.current.run(mode: .default, before: Date(timeIntervalSinceNow: 0.05)) )
+        //
```

Comment:
Did you intend to add a comment here that you forgot?

Reply:
Remove stray comment, update self.runLoop to enable earlier wakeup from manager thread.